lute/std/libs/test.luau

```diff
@@ -74,6 +75,10 @@ local function printFailedTest(failed: FailedTest)
 	print(`❌ {failed.test}`)
 	print(`   {failed.filename}:{failed.linenumber}`)
 	print(`   	{failed.assertion}: {failed.error}`)
+	if failed.stacktrace and failed.stacktrace ~= "" then
```

Comment:
```suggestion
	if failed.stacktrace ~= "" then
```

if the `stacktrace` is always present and just empty string when not set, we don't need to check if it's present